app/containers/Sessions/containers/QuestionListGroup/index.js

```diff
@@ -134,49 +153,121 @@ const QuestionListGroup = ({
         onShowImg={arrowUp}
         imgWithBackground
         label={
-          <Row align="center" justify="between" width="100%" mr={10}>
-            <Box display="flex" align="center">
-              {manage && isManageableGroup && (
-                <Checkbox
-                  id={`group-to-select-${id}`}
-                  mr={8}
-                  onChange={(_, event) => {
-                    event.stopPropagation();
-                    event.preventDefault();
-                    toggleGroup(questions);
-                  }}
-                  checked={checkSelectedGroup(questions)}
-                />
-              )}
-              <StyledInput
-                px={12}
-                py={6}
-                value={title}
-                fontSize={18}
-                fontWeight="bold"
-                placeholder={formatMessage(messages.groupPlaceholder)}
-                width="100%"
-                maxWidth="initial"
-                onBlur={(val) => changeGroupName(val, sessionId, id)}
-                onFocus={selectInputText}
-                disabled={!editingPossible}
-              />
-            </Box>
-            {isDraggableGroup && (
-              <Img
-                src={reorderIcon}
-                disabled={!editingPossible}
-                aria-label={formatMessage(globalMessages.dragHandle)}
-                title={formatMessage(globalMessages.dragHandle)}
-                {...providedGroupDraggable.dragHandleProps}
-              />
-            )}
-          </Row>
+          <div style={{ width: 275 }}>
```

Comment:
We used to do that instead of row styles in the template
```suggestion
          <Box width="275">
```